### DISCUSSION TITLE
Error after updating MOOSE on Windows

### DISCUSSION THREAD

**Discussion by tstepan333** [idaholab/moose#19162] [Q&A Getting Started] [answered]
Hello,

I updated MOOSE and then tried to recompile makefile, but I got the error:

`In file included from /home/tstepan/projects/moose/framework/build/unity_src/auxkernels_Unity.C:50:
/home/tstepan/projects/moose/framework/src/auxkernels/SolutionAux.C: In member function 'virtual libMesh::Real SolutionAux::computeValue()':
/home/tstepan/projects/moose/framework/src/auxkernels/SolutionAux.C:97:63: error: 'const class libMesh::Elem' has no member named 'vertex_average'
   97 |       output = _solution_object.pointValue(_t, _current_elem->vertex_average(), _var_name);
      |                                                               ^~~~~~~~~~~~~~
In file included from /home/tstepan/projects/moose/framework/build/unity_src/auxkernels_Unity.C:52:
/home/tstepan/projects/moose/framework/src/auxkernels/SpatialUserObjectAux.C: In member function 'virtual libMesh::Real SpatialUserObjectAux::computeValue()':
/home/tstepan/projects/moose/framework/src/auxkernels/SpatialUserObjectAux.C:41:53: error: 'const class libMesh::Elem' has no member named 'vertex_average'
   41 |     return _user_object.spatialValue(_current_elem->vertex_average());
      |                                                     ^~~~~~~~~~~~~~
Compiling C++ (in opt mode) /home/tstepan/projects/moose/framework/build/unity_src/fviks_Unity.C...
Compiling C++ (in opt mode) /home/tstepan/projects/moose/framework/build/unity_src/fvkernels_Unity.C...
Compiling C++ (in opt mode) /home/tstepan/projects/moose/framework/build/unity_src/geomsearch_Unity.C...
make: *** [/home/tstepan/projects/moose/framework/build.mk:144: /home/tstepan/projects/moose/framework/build/unity_src/auxkernels_Unity.x86_64-conda-linux-gnu.opt.lo] Error 1
make: *** Waiting for unfinished jobs....
In file included from /home/tstepan/projects/moose/framework/build/unity_src/geomsearch_Unity.C:3:
/home/tstepan/projects/moose/framework/src/geomsearch/FindContactPoint.C: In function 'void Moose::findContactPoint(PenetrationInfo&, libMesh::FEBase*, libMesh::FEBase*, libMesh::FEType&, const libMesh::Point&, bool, libMesh::Real, bool&)':
/home/tstepan/projects/moose/framework/src/geomsearch/FindContactPoint.C:104:44: error: 'const class libMesh::Elem' has no member named 'vertex_average'
  104 |         dim - 1, fe_side_type, side, side->vertex_average(), TOLERANCE, false);
      |                                            ^~~~~~~~~~~~~~
In file included from /home/tstepan/projects/moose/framework/build/unity_src/fvkernels_Unity.C:3:
/home/tstepan/projects/moose/framework/src/fvkernels/FVBodyForce.C: In member function 'virtual ADReal FVBodyForce::computeQpResidual()':
/home/tstepan/projects/moose/framework/src/fvkernels/FVBodyForce.C:45:78: error: 'const class libMesh::Elem' has no member named 'vertex_average'
   45 |   Real factor = _scale * _postprocessor * _function.value(_t, _current_elem->vertex_average());
      |                                                                              ^~~~~~~~~~~~~~
make: *** [/home/tstepan/projects/moose/framework/build.mk:144: /home/tstepan/projects/moose/framework/build/unity_src/geomsearch_Unity.x86_64-conda-linux-gnu.opt.lo] Error 1
make: *** [/home/tstepan/projects/moose/framework/build.mk:144: /home/tstepan/projects/moose/framework/build/unity_src/fvkernels_Unity.x86_64-conda-linux-gnu.opt.lo] Error 1`

To update your MOOSE repository I used the following commands:

cd ~/projects/moose
git fetch origin
git rebase origin/master

Could you help me?

**Comment by GiudGiud** [comment, marked as answer]
Hello

you forgot to update libmesh. 
Is this a conda install? If so just run `conda update --all`
Probably will be good to `make clobberall` before running make again too

Guillaume

**Reply by tstepan333** [reply]
Thanks, it helped
